package.json

```diff
@@ -155,7 +155,7 @@
       "src/test.js"
     ],
     "moduleNameMapper": {
-      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/test/__mocks__/fileMock.js",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)\\??$": "<rootDir>/test/__mocks__/fileMock.js",
```

Comment:
We were already telling `jest` to mock imports for these file types. This tells `jest` to keep mocking them even if they include an optional question mark at the end (`\??`, but with an extra layer of escaping).